src/docfx/lib/moniker/MonikerRangeParser.cs

```diff
@@ -10,6 +10,9 @@ namespace Microsoft.Docs.Build
 {
     internal class MonikerRangeParser
     {
+        public readonly MonikerComparer MonikerAscendingComparer;
+        public readonly MonikerComparer MonikerDescendingComparer;
```

Comment:
No need to put these in parser? We can create an instance from parse when need it.

Reply:
Or put _monikerOrder into comparaer, so MonikerRangeParser and comparaer need not depend on each other

---
In reply to: [235620698](https://github.com/dotnet/docfx/pull/3655#discussion_r235620698) [](ancestors = 235620698)